lib/pages/chat/chat_pinned_events/pinned_messages.dart

```diff
@@ -44,17 +46,21 @@ class PinnedMessagesController extends State<PinnedMessages>
   List<String> get selectedPinnedEventsIds =>
       selectedEvents.value.map((event) => event.eventId).toList();
 
+  bool get isCurrentUserAdmin =>
+      room!.isUserAdmin(Matrix.of(context).client.userID);
+
   List<ContextMenuItemModel> get pinnedMessagesActionsList => [
-        ContextMenuItemModel(
-          text: L10n.of(context)!.unpin,
-          imagePath: ImagePaths.icUnpin,
-          color: Theme.of(context).colorScheme.onSurface,
-          onTap: ({extra}) async {
-            if (extra is Event) {
-              unpin(extra.eventId);
-            }
-          },
-        ),
+        if (isCurrentUserAdmin)
+          ContextMenuItemModel(
+            text: L10n.of(context)!.unpin,
+            imagePath: ImagePaths.icUnpin,
+            color: Theme.of(context).colorScheme.onSurface,
+            onTap: ({extra}) async {
+              if (extra is Event) {
+                unpin(extra.eventId);
+              }
+            },
+          ),
```

Comment:
let user try to unpin and show that they don't have permission, because we don't know for sure what people(not only admin) have permission to unpin or pin